kubernetes/samples/scripts/create-weblogic-domain/create-weblogic-credentials.sh

```diff
@@ -51,13 +64,22 @@ while getopts "hu:p:n:d:" opt; do
     ;;
     d) domainUID="${OPTARG}"
     ;;
+    s) secretName="${OPTARG}"
+    ;;
     h) usage 0
     ;;
     *) usage 1
     ;;
   esac
 done
-secretName=$domainUID-weblogic-credentials
+
+if [ -z $secretName ]; then
+  if [ -z $domainUID ]; then
+    secretName=weblogic-credentials
```

Comment:
The comment earlier says the default secretName would be domain1-weblogic-credentials when neither secretName or domainUID are provided

Reply:
The script initializes the domainUID to domain1. So if the domainUID is not set here, which means that it is overridden by the -d option with "". So we now fall into case #4.

Reply:
hmmm, maybe I need to check if the domainUID is not to ""? I tested all permutations. Let me test again.

Reply:
case #1: 
bash-4.2$ sh create-weblogic-credentials.sh -u weblogic -p welcome1 -d domain40 -s mysecret
secret "mysecret" created
secret "mysecret" labeled
The secret mysecret has been successfully created in the default namespace.

case #2:
bash-4.2$ sh create-weblogic-credentials.sh -u weblogic -p welcome1
secret "domain1-weblogic-credentials" created
secret "domain1-weblogic-credentials" labeled
The secret domain1-weblogic-credentials has been successfully created in the default namespace.

Case #3:
bash-4.2$ sh create-weblogic-credentials.sh -u weblogic -p welcome1 -d domain40
secret "domain40-weblogic-credentials" created
secret "domain40-weblogic-credentials" labeled

case #4:
bash-4.2$ sh create-weblogic-credentials.sh -u weblogic -p welcome1 -d ""
secret "weblogic-credentials" created
The secret weblogic-credentials has been successfully created in the default namespace.

Reply:
Thanks for clarifying! 

Do we support "" as valid domainUID? 

Do you think the comment for #4 should be updated to 
4) weblogic-credentials - if secretName is not specified, and domainUID is specified as "".

Reply:
Good catch on a copy-and-paste error.